### PR TITLE
Create env measurement

### LINKED ISSUE #14: Create Garden EnvMeasurement Endpoint
Request Format:
```
POST /api/v1/gardens/:id/env_measurements
Content-Type: application/json
Accept: application/json

 {
        "soil_temperature": 55.52,
        "soil_moisture": 85.33,
 }
```

Response Format:
```
status: 201
body:
{"created_at"=>"2019-09-04T20:59:59.110Z",
"soil_moisture"=>85.33,
"soil_temperature"=>55.52}
```

### Checks
- [x] For success, returns status code 201
- [x] Returns correct response format json

Sad path
- [x] For error, returns status code 404

### PR DESCRIPTION
**Pull request by bplantico**
## Changes proposed in this pull request:
* Adds POST request to api/v1/gardens/:id/env_measurements which takes soil_temperature and soil_moisture parameters and creates record in env_measurements table.

## Card(s) closed in this pull request:
close #14 

## What did you struggle on to complete?
* Didn't struggle on anything, this is a pretty straightforward create action. We're also not validating presence of soil_temperature or soil_moisture which made this easier since it didn't require extra logic in case either or both of those were missing.

### Test Coverage Percentage: 100%
- [x] Tested my new feature(s) as well as any feasible edge cases (if possible)
- [x] Checked coverage/index.html - did not add any new code that's not covered by testing (if possible)
- [x] Ran the test suite - all tests are passing
- [x] Merged in the latest master to my branch with git pull origin master & resolved merge conflicts
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: app/models/env_measurement.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class EnvMeasurement < ApplicationRecord
 	belongs_to :garden
+
+	validates_numericality_of :soil_temperature
+	validates_numericality_of :soil_moisture, greater_than_or_equal_to: 0
```
Comment (Mycobee) [inline app/models/env_measurement.rb]: Good thinking preventing the negative percentage.  I also like the numerical temperature validation.

File: pull_request_template.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ close #
 *
 
 ## Current Test Suite:
-### Test Coverage Percentage: x%
```
Comment (Mycobee) [inline pull_request_template.md]: I like adding model coverage in here to ensure good unit testing.
